cloudbuild.yaml

```diff
@@ -88,8 +105,17 @@ steps:
             --build-arg=WEIGHTS_FILE=yolov7-tiny.pt \
             --build-arg=CLASS_NAMES_FILE=coco.names \
             --platform=linux/amd64,linux/arm64 \
-            --label "git-commit=${SHORT_SHA}" \
             --push \
+            --annotation 'index:teknoir.org/display-name=yolov7-vanilla' \
+            --annotation 'index:teknoir.org/description="Yolo v7 CPU vanilla."' \
+            --annotation 'index:teknoir.org/framework=pythorch' \
+            --annotation 'index:teknoir.org/image-type=model' \
+            --annotation 'index:teknoir.org/model-type-description=yolov7' \
+            --annotation 'index:teknoir.org/model-name=vanilla' \
+            --annotation 'index:teknoir.org/model-dataset=coco' \
+            --annotation 'index:teknoir.org/version=v1.0.0' \
+            --annotation 'index:teknoir.org/model-feature=cpu' \
+            --label "git-commit=${SHORT_SHA}" \
             -t us-central1-docker.pkg.dev/${PROJECT_ID}/teknoir-ai/yolov7-vanilla:cpu-latest \
```

Comment:
If these are not accessible between namespaces (teknoir-ai) we might need to look into creating a shared image repository in the future for all namespaces (at least for the core builds)